neon_utils/packaging_utils.py

```diff
@@ -265,3 +266,33 @@ def get_skill_license():  # TODO: Implement OSM version of this
     skill_data = dict_merge(default_skill, skill_data)
     skill_data["requirements"]["python"].sort()
     return dict(dict_merge(skill_data, readme_data))
+
+def install_packages_from_pip(module: str, packages: List[str]) -> int:
+    """
+    Install a Python package using pip
+    :param module: string neon module to install dependency for
```

Comment:
One more small change, can this be refactored to `core_module` or `constraints_module` to clarify that it's not a module being installed